### PR TITLE
hw-1_initial

### PR DESCRIPTION
**Pull request by quantum-noise**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "git+https://github.com/quantum-noise/njm.git"
+  },
+  "author": "Aliaksandr Klimau",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/quantum-noise/njm/issues"
+  },
+  "homepage": "https://github.com/quantum-noise/njm#readme",
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
```
Comment (boccob) [inline package.json]: But you can't run it in production without babel. Why is it in `devDependencies`?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "babel-eslint": "^7.2.3",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (boccob) [inline package.json]: I'd suggest using `babel-preset-env` with `target: node`.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "babel-preset-stage-2": "^6.24.1",
+    "babel-register": "^6.26.0",
```
Comment (boccob) [inline package.json]: Do you really need it?

File: package.json (hunk 1 above)
Comment (quantum-noise) [reply package.json]: But what if I add build script for production in the future?

File: package.json (hunk 2 above)
Comment (quantum-noise) [reply package.json]: Great idea!

File: package.json (hunk 3 above)
Comment (quantum-noise) [reply package.json]: Maybe not =)

File: package.json (hunk 1 above)
Comment (boccob) [reply package.json]: Then you'll move babel to devDependencies :P

**Pull request merged**